config/config.go

```diff
@@ -52,6 +55,7 @@ const (
 	"ipfs_end_point": "http://localhost:5002/", 
 	"organization_name": "ExampleOrganization", 
 	"passthrough_enabled": false,
+	"rate_limit": 20,
```

Comment:
Is 20 request per minute enough? I would set some bigger value to not limiting users by default, may be even unlimited rate.

Reply:
Thansk @vsbogd , then lets remove the  "rate_limit": 20 in the json , we default to infinity in the code and this property becomes optional in its true sense. Burst size will have no meaning when the rate is infinity , we will explain this in the documentation at README.md

Reply:
lets also remove the burst_size and default it to math.MaxInt64, from the config json.